17 error Error: Command failed with exit code 1: git remote update
17 error error: cannot lock ref 'refs/remotes/origin/gh-pages': is at 8f2e7bd833fdef3705b88fc741cd2580f47ed480 but expected 9ba0fee83e12e79175853a4e47541f9f4da23538
17 error From github.com:crosswise-finance/crosswise-toolkit
17 error  ! 9ba0fee..8f2e7bd  gh-pages   -> origin/gh-pages  (unable to update local ref)
17 error error: Could not fetch origin
17 error Fetching origin
17 error     at makeError (D:\tutorial\blockchain\CrossWise\crosswise-toolkit\node_modules\execa\lib\error.js:59:11)
17 error     at Function.module.exports.sync (D:\tutorial\blockchain\CrossWise\crosswise-toolkit\node_modules\execa\index.js:186:17)
17 error     at Object.execSync (D:\tutorial\blockchain\CrossWise\crosswise-toolkit\node_modules\@lerna\child-process\index.js:39:16)
17 error     at updateRemote (D:\tutorial\blockchain\CrossWise\crosswise-toolkit\node_modules\@lerna\version\lib\is-behind-upstream.js:34:16)
17 error     at isBehindUpstream (D:\tutorial\blockchain\CrossWise\crosswise-toolkit\node_modules\@lerna\version\lib\is-behind-upstream.js:16:3)
17 error     at VersionCommand.initialize (D:\tutorial\blockchain\CrossWise\crosswise-toolkit\node_modules\@lerna\version\index.js:153:9)
